nUpdate.Shared/Updating/UpdateResult.cs

```diff
@@ -47,6 +47,44 @@ internal class UpdateResult
                         config.Architecture == Architecture.X64 && !is64Bit)
                         continue;
 
+
+
+                    if (config.RolloutConditions != null && config.RolloutConditions.Count != 0)
+                        if (conditions == null || !conditions.Any())
```

Comment:
I would write `condition.Count == 0` that's better to understand.

Reply:
Any can also used to look there is an entry, but you are right. It is better to use 'conditions?.Count = 0'.
I will edit this.

Reply:
@FireEmerald It's also okay to use `Any` for checking, if there are elements in a collection. I mean, it is actually the same but you do not have a predicate.